File: hangman/app/src/main/java/lab1/hangman/SettingsActivity.java
package lab1.hangman;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;

public class SettingsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getSupportFragmentManager()
                .beginTransaction()
                .replace(android.R.id.content, new SettingsFragment())
                .commit();
    }

    public static void start(Context context) {
        Intent starter = new Intent(context, SettingsActivity.class);
        context.startActivity(starter);
    }
}

File: mediaplayer/app/src/main/java/com/hbvhuwe/mediaplayer/ui/SettingsActivity.kt
package com.hbvhuwe.mediaplayer.ui

import android.content.Context
import android.content.Intent
import android.os.Bundle
import com.hbvhuwe.mediaplayer.R
import kotlinx.android.synthetic.main.activity_settings.*


class SettingsActivity : ThemedActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_settings)

        toolbar.setTitle(R.string.settings_title)

        setSupportActionBar(toolbar)
        supportActionBar?.setDisplayHomeAsUpEnabled(true)
    }

    companion object {

        fun start(context: Context) {
            val starter = Intent(context, SettingsActivity::class.java)
            context.startActivity(starter)
        }
    }
}